benchmarks/inference-server/jetstream/jetstream.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         cloud.google.com/gke-tpu-accelerator: tpu-v5-lite-podslice
       containers:
       - name: maxengine-server
-        image: us-docker.pkg.dev/cloud-tpu-images/inference/maxengine-server:v0.2.0
+        image: us-docker.pkg.dev/cloud-tpu-images/inference/maxengine-server:v0.2.2
```

Comment:
Since you've added the TF templates for other resources can you convert the deployment to a TF template as well to keep it consistent?

Reply:
Added it in as the `maxengine` module